doc/install.md

```diff
@@ -0,0 +1,77 @@
+# Install EDL dependencies
+
+## Kubernetes
+
+Before everything, make sure you have a running Kubernetes v1.7.* cluster and a working `kubectl`.
+You may prepare a Kubernetes on your laptop using [minikube](#minikube), create a Kubernetes Cluster on [AWS](#AWS)
+or any other way as you expediently.
+
+### kubectl
+
+`kubectl` is the CLI tool to manager your Kubernetes cluster, and kubectl `v1.9.0` works well for Kubernets v1.7.*,
+you can install it by following commands:
+
+- MacOS
+
+```bash
+wget -O kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/darwin/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
+- Linux
```

Comment:
How about telling readers that if they are using windows, use a docker environment to achieve this?

Reply:
The user can achieve a binary file adapted windows, will add the section.